community-translations.md

```diff
@@ -0,0 +1,3 @@
+# Community Course Translations
+
+We are grateful for the contributions of the community in translating this course. Although the OpenSauced team does not currently maintain these projects, we want to share the forked copy of the repositories with those taking the course. Below, you will find the latest community translations.
```

Comment:
Can you please add "Intro to Open Source" in the first sentence for clarity?

You can simply click the "Commit" button below this suggestion to accept and commit the changes. :)

```suggestion
We are grateful for the contributions of the community in translating this Intro to Open Source course. Although the OpenSauced team does not currently maintain these projects, we want to share the forked copy of the repositories with those taking the course. Below, you will find the latest community translations.
```